src/encdatavault_fmt_plug.c

```diff
@@ -0,0 +1,411 @@
+/*
+ * Cracker for ENCSecurity Data Vault.
+ *
+ * This software is Copyright (c) 2021 Sylvain Pelissier sylvain.pelissier@kudelskisecurity.com
+ * and it is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_encdadatavault;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_encdadatavault);
+#else
+
+#include <string.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#include "aes.h"
+#include "arch.h"
+#include "md5.h"
+#include "misc.h"
+#include "common.h"
+#include "formats.h"
+#include "params.h"
+#include "options.h"
+#include "base64_convert.h"
+#include "johnswap.h"
+#include "simd-intrinsics.h"
+
+#define FORMAT_TAG              "$encdv$"
+#define FORMAT_TAG_LENGTH       (sizeof(FORMAT_TAG) - 1)
+#define FORMAT_LABEL            "ENCDataVault"
+#define FORMAT_NAME             ""
+#define ALGORITHM_NAME          "MD5 " MD5_ALGORITHM_NAME
```

Comment:
The `MD5_ALGORITHM_NAME` resolvs to `AVX2 8x3` on my gear. That's not correct since there's no SIMD support.